### PR TITLE
Income and age tests

### PR DESCRIPTION
**Pull request by Hjortsberg**
Please check if my tests do test the intended logic.

### CODE REVIEW

**Comment by willeponken** [comment]
Fyfan vad dåligt PR-namn!!½!!!!

**Comment by willeponken** [comment]
FIX PL0X

**Comment by Hjortsberg** [comment]
Sorry, what is that language?

**Comment by willeponken** [comment]
English

**Comment by willeponken** [comment]
2018-10-03-153425_1566x727_scrot
WHAT IS THIS

**Comment by willeponken** [comment]
PLEASE ADD `production/` TO `.gitignore`!!!!

File: test/PaymentTest.java [hunk 1]
```diff
@@ -14,10 +14,41 @@ public void testSilly()
         assertEquals(1, 1);
     }
 
+
+    @Test
+    public void loanAndSubsidyUnderTwentyZero() throws IOException {
+        PaymentImpl payment = new PaymentImpl(getCalendar());
+        long amount = payment.getMonthlyAmount("20001201-4509",0, 100, 100);
```
Comment (willeponken) [inline test/PaymentTest.java]: long -> int and fix spaces plz

File: test/PaymentTest.java [hunk 2]
```diff
@@ -14,10 +14,41 @@ public void testSilly()
...
+        assertEquals(0, amount);
+    }
+
+    @Test
+    public void loanAndSubsidyOverFiftySix() throws IOException {
+        PaymentImpl payment = new PaymentImpl(getCalendar());
+        long amount = payment.getMonthlyAmount("19591201-4509",0,100, 100);
```
Comment (willeponken) [inline test/PaymentTest.java]: long -> int and fix spaces plz

File: test/PaymentTest.java [hunk 3]
```diff
@@ -14,10 +14,41 @@ public void testSilly()
...
         PaymentImpl payment = new PaymentImpl(getCalendar());
-        payment.getNextPaymentDay();
+        long amount = payment.getMonthlyAmount("19681201-4509",0,100, 100);
+        assertEquals(2816, amount);
     }
 
+    @Test
+    public void subsidyFulltimeEarnings() throws IOException{
+        PaymentImpl payment = new PaymentImpl(getCalendar());
+        long amount = payment.getMonthlyAmount("19901201-4509",85814,100, 100);
```
Comment (willeponken) [inline test/PaymentTest.java]: long -> int and fix spaces plz

File: test/PaymentTest.java [hunk 4]
```diff
@@ -14,10 +14,41 @@ public void testSilly()
...
+        assertEquals(0, amount);
+    }
+
+    @Test
+    public void subsidyParttimeEarnigs() throws IOException{
+        PaymentImpl payment = new PaymentImpl(getCalendar());
+        long amount = payment.getMonthlyAmount("19901201-4509",128723,75, 100);
```
Comment (willeponken) [inline test/PaymentTest.java]: long -> int and fix spaces plz

File: test/PaymentTest.java [hunk 5]
```diff
@@ -9,15 +9,37 @@
...
+
+    @Test
+    public void loanAndSubsidyOverFiftySix() throws IOException {
+        PaymentImpl payment = new PaymentImpl(getCalendar());
+        int amount = payment.getMonthlyAmount("19591201-4509", 0, 100, 100);
+        assertEquals(0, amount);
+    }
+
+    @Test
+    public void subsidyOverFortySeven() throws IOException{
```
Comment (willeponken) [inline test/PaymentTest.java]: Missing space `throws IOException{`

File: test/PaymentTest.java [hunk 6]
```diff
@@ -9,15 +9,37 @@
...
+        PaymentImpl payment = new PaymentImpl(getCalendar());
+        int amount = payment.getMonthlyAmount("19681201-4509", 0, 100, 100);
+        assertEquals(2816, amount);
     }
 
     @Test
-    public void evalGetLoan() throws IOException {
+    public void subsidyFullTimeEarnings() throws IOException{
```
Comment (willeponken) [inline test/PaymentTest.java]: Same here: `throws IOException{`

File: test/PaymentTest.java [hunk 7]
```diff
@@ -9,15 +9,37 @@
...
         PaymentImpl payment = new PaymentImpl(getCalendar());
-        payment.getNextPaymentDay();
+        int amount = payment.getMonthlyAmount("19901201-4509", 85814, 100, 100);
+        assertEquals(0, amount);
     }
 
+    @Test
+    public void subsidyPartTimeEarnings() throws IOException{
```
Comment (willeponken) [inline test/PaymentTest.java]: `throws IOException{`!!!!

File: test/PaymentTest.java (hunk 5 above)
Comment (willeponken) [reply test/PaymentTest.java]: `IOException{` -> `IOException {`

**Pull request merged**